test_data/makedata_suites/060_computed_values.js

```diff
@@ -0,0 +1,776 @@
+  ];
+
+  // Expected output array for the TestView query output 
+  let index_exp_output = [64000, 64000, 64000, 64000, 64000, 64000, 0, 0, 0, 0, 64000, 0, 11, 64000, 17, 32000, 33, 64000, 64000, 1, 64000];
+
+  return [index_array, index_exp_output];
+}
+
+// this function will provide all the queries for views
+function viewsArray(dbCount) {
+  // get all the view's variable name from the variable_name_declaration method globally
+  let view = views_name_declaration(dbCount);
+  
+  let views_array = [
+    `for doc in ${view[0]} search doc.cv_field == SOUNDEX('sky') collect with count into c return c`,
+    `for doc in ${view[0]} search doc.cv_field == SOUNDEX('dog') collect with count into c return c`,
+    `for doc in ${view[0]} search doc.cv_field_insert == SOUNDEX('frog') collect with count into c return c`,
+    `for doc in ${view[0]} search doc.cv_field_update == SOUNDEX('beer') collect with count into c return c`,
+    `for doc in ${view[0]} search doc.cv_field_replace == SOUNDEX('water') collect with count into c return c`,
+    `for doc in ${view[0]} filter doc.cv_field == to_hex(doc.name) collect with count into c return c`,
+    `for doc in ${view[0]} filter doc.cv_field == CONCAT('42_', TO_STRING(doc.field)) collect with count into c return c`,
+    `for doc in ${view[0]} search doc.cv_field1=='foo' and doc.cv_field2=='bar' and doc.cv_field3=='baz' collect with count into c return c`,
+    `for doc in ${view[0]} filter doc.cv_field == CONCAT(doc._key, ' ', doc._id, ' ', doc._rev) collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field == SOUNDEX('sky') collect with count into c return c`, 
+    `for doc in ${view[1]} search doc.cv_field == SOUNDEX('dog') collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field_insert == SOUNDEX('frog') collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field_update == SOUNDEX('beer') collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field_replace == SOUNDEX('water') collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field == null collect with count into c return c`,
+    `for doc in ${view[1]} filter doc.cv_field == to_hex(doc.name) collect with count into c return c`,
+    `for doc in ${view[1]} filter doc.cv_field == CONCAT('42_', TO_STRING(doc.field)) collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field1=='foo' and doc.cv_field2=='bar' and doc.cv_field3=='baz' collect with count into c return c`,
+    `for doc in ${view[1]} filter doc.cv_field == CONCAT(doc._key, ' ', doc._id, ' ', doc._rev) collect with count into c return c`
+  ];
+
+  // Expected output array for the TestView views query output
+  let views_exp_output = [64000, 64000, 64000, 0, 0, 64000, 96000, 64000, 64000, 64000, 64000, 64000, 0, 0, 64000, 64000, 96000, 64000, 64000];
+
+  return [views_array, views_exp_output];
```

Comment:
Lets store array of tuples in following way:
[
[<query_1>, <expected_output_1>],
[<query_2>, <expected_output_2>],
.....
]

Reply:
Array to tuple conversion done.